rslib/src/search/sqlwriter.rs

```diff
@@ -514,6 +579,35 @@ impl SqlWriter<'_> {
         Ok(())
     }
 
+    #[allow(clippy::type_complexity)]
```

Comment:
Rather than silencing this warning, it would be nicer to create a struct for the return value.